README.md

```diff
@@ -136,6 +136,7 @@ To customize your workflow, you can set the following parameters in a [`run-test
 | `api_key`                 | env var name | `DATADOG_API_KEY`                         | The name of the environment variable containing the API key.                                                                                                                                                            |
 | `app_key`                 | env var name | `DATADOG_APP_KEY`                         | The name of the environment variable containing the APP key.                                                                                                                                                            |
 | `background`              | boolean      | `false`                                   | Whether or not this step should run in the background. [See official CircleCI documentation](https://circleci.com/docs/configuration-reference/#run).                                                                   |
+| `batch_timeout`           | number       | _30 minutes_                              | The duration (in milliseconds) after which the batch fails as timed out. The default is 30 minutes.                                                                                                                     |
```

Comment:
```suggestion
| `batch_timeout`           | number       | _30 minutes_                              | The specified amount of time (in milliseconds) to wait for a batch before timing out. The default value is 30 minutes.                                                                                                                     |
```

Reply:
Hey for this we actually initially asked in #documentation [here](https://dd.slack.com/archives/C0DESMBQU/p1722245368197199)
would you still like me to change it?

Reply:
Hi @teodor2312, you can ignore my suggestion on this. Thanks!